ui/src/config/values.ts

```diff
@@ -897,3 +897,32 @@ export const HTMLStyles = `
     })
     .filter((i) => !!i)
     .join(' ');
+
+export const HTMLStylesTiptapEditor = `
+    custom-table-tiptap-editor
+    prose
+    w-full
+    max-w-none
+    text-grey-800
+    transition-colors 
+    duration-500 
+    prose-headings:font-medium 
+    prose-headings:font-montserrat 
+    prose-headings:text-grey-800 
+    prose-a:text-grey-800 
+    prose-a:underline 
+    prose-a:decoration-teal-500 
+    prose-a:decoration-2 
+    prose-a:underline-offset-1 
+    prose-a:outline-none 
+    focus:prose-a:bg-yellow-300 
+    prose-blockquote:border-teal-500 
+    prose-blockquote:text-grey-600
+    prose-strong:text-grey-800
+`
+    .split(' ')
+    .map((i) => {
+        return i.replace(/\n/g, '').length ? i.replace(/\n/g, '') : undefined;
+    })
+    .filter((i) => !!i)
```

Comment:
Is the map and filter really needed for HTMLStyles and HTMLStylesTiptapEditor?

Reply:
I can remove the filter without error and still achieves the same result, otherwise when the map is gone it errors with: 
```
InvalidCharacterError: DOMTokenList.add: The token can not contain whitespace.
```

Reply:
Did you try .split('\n').join(' ')?

Reply:
That works, is it work removing the filter/map from HTMLStyles as well, since from what I understand, that's what it's attempting to achieve anyway, albiet out of the scope of this ticket?

Reply:
Yes, there are lots of bits of code like this that can be improved on the fly when we're working in a specific area and we see them.

Reply:
split/join does however leave unnecessary whitespace in the resulting string so isn't as neat, a reaplce is slighlt better but leaves a leading space:

.replaceAll(/\s+/g,' ')

Reply:
the whitespace actually makes no difference in this case